app/src/main/java/com/ncnf/map/Marker.java

```diff
@@ -5,21 +5,26 @@
 
 import com.google.android.gms.maps.model.LatLng;
 import com.google.maps.android.clustering.ClusterItem;
+import com.ncnf.event.Event;
+
+import java.util.List;
 
 public class Marker implements ClusterItem {
 
     private final LatLng position;
     private final String title;
     private final String snippet;
+    private final List<Event> eventList;
 
-    public Marker(float lat, float lon, String title, String snippet){
-        this(new LatLng(lat, lon), title, snippet);
+    public Marker(float lat, float lon, String title, String snippet, List<Event> eventList){
+        this(new LatLng(lat, lon), title, snippet, eventList);
     }
 
-    public Marker(LatLng position, String title, String snippet){
+    public Marker(LatLng position, String title, String snippet, List<Event> eventList){
```

Comment:
With your previous implementation you would have add an list attribute which would have been null, create an empty list instead and check for nullity when receiving argument